_includes/codepens/readonly-demo/index.html

```diff
@@ -0,0 +1,26 @@
+<script type="text/javascript">
+  function getEditorStatus (editorId) {
+    return tinymce.get(editorId).mode.get();
+  };
+  function setEditorStatus (editorId, currentStatus) {
+    if (currentStatus === "design") {
+      tinymce.get(editorId).mode.set("readonly");
+    } else {
+      tinymce.get(editorId).mode.set("design");
+    }
+  };
+  function enableDisable (targetEditor,targetElement) {
+    var status = getEditorStatus (targetEditor);
+    var button = document.getElementById(targetElement);
+    setEditorStatus (targetEditor, status);
+    if (status === "design") {
+      button.innerText="Enable editor";
```

Comment:
Just a style thing, it'd be more readable if you add spaces around the equals. eg:
```
button.innerText = "Enable editor";
```

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1601/commits/58954c69ece8380bc72ca0e167819bd224c0751e